.circleci/config.yml

```diff
@@ -993,92 +960,61 @@ jobs:
       - *setup_env
       - run:
           name: "Deploy to dockerhub"
-          command: yarn-project/deploy_dockerhub.sh
-
-  # build-deployment-canary:
-  #   machine:
-  #     image: ubuntu-2204:2023.07.2
-  #   resource_class: large
-  #   steps:
-  #     - *checkout
-  #     - *setup_env
-  #     - run:
-  #         name: "Build"
-  #         command: build canary true
-
-  # canary-uniswap-test:
-  #   machine:
-  #     image: ubuntu-2204:2023.07.2
-  #   resource_class: large
-  #   steps:
-  #     - *checkout
-  #     - *setup_env
-  #     - run:
-  #         name: "Test"
-  #         command: run_script canary ./scripts/run_tests ./src/uniswap_trade_on_l1_from_l2.test.ts canary ./scripts/docker-compose.yml
-
-  # canary-browser-test:
-  #   machine:
-  #     image: ubuntu-2204:2023.07.2
-  #   resource_class: large
-  #   steps:
-  #     - *checkout
-  #     - *setup_env
-  #     - run:
-  #         name: "Test"
-  #         command: run_script canary ./scripts/run_tests ./src/aztec_js_browser.test.ts canary ./scripts/docker-compose-browser.yml
-
-  # canary-cli-test:
-  #   machine:
-  #     image: ubuntu-2204:2023.07.2
-  #   resource_class: large
-  #   steps:
-  #     - *checkout
-  #     - *setup_env
-  #     - run:
-  #         name: "Test"
-  #         command: run_script canary ./scripts/run_tests ./src/cli.test.ts canary ./scripts/docker-compose.yml
+          command: |
+            [ -n "$COMMIT_TAG" ] || exit 0
+            deploy_dockerhub noir x86_64,arm64
+            deploy_dockerhub pxe x86_64,arm64
```

Comment:
pxe image has been removed as it can now be started from the aztec-sandbox image with `MODE=pxe`